boot/freeldr/freeldr/ntldr/winldr.c

```diff
@@ -1251,6 +1251,29 @@ LoadAndBootWindowsCommon(
     /* "Stop all motors", change videomode */
     MachPrepareForReactOS();
 
+    /* Show the "debug mode" notice if needed */
+    BOOLEAN DebugMode = !NtLdrGetOption(BootOptions, "NODEBUG") && /* NODEBUG is absent, priority over DEBUG option */
+                        !NtLdrGetOption(BootOptions, "CRASHDEBUG") && /* CRASHDEBUG is abset, no wait during the boot */
+                        !!NtLdrGetOption(BootOptions, "DEBUG"); /* DEBUG is present */
+    if (DebugMode)
+    {
+        /* It is booting in debug mode */
+        TuiPrintf("Booting in debug mode\n");
```

Comment:
```suggestion
        TuiPrintf("Debug mode requested.\n");
```

Reply:
@HBelusca What do you think? I prefer "Booting in debug mode" message.